python/plugins/MetaSearch/util.py

```diff
@@ -61,7 +62,9 @@ def __init__(self):
         """init"""
         self.ppath = os.path.dirname(os.path.abspath(__file__))
         self.metadata = configparser.ConfigParser()
-        self.metadata.readfp(open(os.path.join(self.ppath, 'metadata.txt')))
+        f = codecs.open(os.path.join(self.ppath, 'metadata.txt'), "r", "utf8")
```

Comment:
For extra points use

    with codecs.open(filename) as f:
        self.metadata.read_file(f)